crates/asm/src/lib.rs

```diff
@@ -190,42 +193,31 @@ mod tests {
         let ops: Vec<Op> = vec![
             Stack::Push(0x1234567812345678).into(),
             Stack::Push(0x0F0F0F0F0F0F0F0F).into(),
-            Stack::Swap.into(),
-            Stack::Dup.into(),
+            StateRead::KeyRange,
+            StateRead::KeyRangeExtern,
         ];
         roundtrip(ops);
     }
 
     #[test]
+    #[allow(clippy::useless_conversion)]
     fn roundtrip_args_end() {
         let ops: Vec<Op> = vec![
-            Stack::Swap.into(),
-            Stack::Dup.into(),
-            Stack::Push(0x0F0F0F0F0F0F0F0F).into(),
-        ];
-        roundtrip(ops);
-    }
-
-    #[test]
-    fn roundtrip_args_interspersed() {
-        let ops: Vec<Op> = vec![
-            Stack::Push(0x1234567812345678).into(),
-            Stack::Swap.into(),
+            StateRead::KeyRange.into(),
+            StateRead::KeyRangeExtern.into(),
             Stack::Push(0x0F0F0F0F0F0F0F0F).into(),
-            Stack::Dup.into(),
-            Stack::Push(0x1234567812345678).into(),
         ];
         roundtrip(ops);
     }
```

Comment:
Why is this removed?

Reply:
Wow I have no idea, good catch! Re-added